local-cli/runAndroid/runAndroid.js

```diff
@@ -92,17 +92,20 @@ function buildAndRun(args, reject) {
   }
 
   try {
-    const packageName = fs.readFileSync(
-      'app/src/main/AndroidManifest.xml',
+    const manifest = fs.readFileSync(
+      'app/build/intermediates/manifests/full/debug/AndroidManifest.xml',
```

Comment:
This location will only work for people only using buildTypes

When using flavors and buildtypes, it will look like this:
`app/build/intermediates/manifests/full/paid/debug/AndroidManifest.xml`

(`debug` = buildType, `paid` = flavor)